include/ibamr/BrinkmanPenalizationStrategy.h

```diff
@@ -98,14 +93,6 @@ class BrinkmanPenalizationStrategy : public SAMRAI::tbox::Serializable
         return d_object_name;
     } // getName
 
-    /*
-     * \brief Get the Brinkman coefficient.
-     */
-    double getBrinkmanCoefficient() const
```

Comment:
Same as above: since this is in the public interface for this strategy class, I'd suggest that we indicate that the interface is deprecated rather than removing it entirely.

Reply:
Changed the function name from `getBrinkmanCoefficient()` to ` getBrinkmanPenaltyFactor()`.